modules/basegl/src/processors/sphererenderer.cpp

```diff
@@ -168,19 +186,26 @@ void SphereRenderer::process() {
                         drawer.draw(MeshDrawerGL::DrawMode::Points);
                     }
                 }
-
                 break;
             }
             case RenderMode::EntireMesh:
                 [[fallthrough]];
             default:  // render all parts of the input meshes as points
-                MeshDrawerGL::DrawObject drawer(mesh->getRepresentation<MeshGL>(),
-                                                mesh->getDefaultMeshInfo());
-                utilgl::setShaderUniforms(shader, *mesh, "geometry");
                 drawer.draw(MeshDrawerGL::DrawMode::Points);
-
                 break;
         }
+
+        // render selection only for first mesh input
+        if (!indices.empty() && (mesh == inport_.getData())) {
+            shader.setUniform("selectionMix", 1.0f);
```

Comment:
I'm not a big fan of this use of mix unless the idea is to have animation. Better to use `if (selected) ...` otherwise